internal/api/shared/schemas/loader.go

```diff
@@ -42,10 +42,44 @@ type cueDefs struct {
 	kindCuePath string
 }
 
+type cueDefsBis struct {
```

Comment:
another name for this struct would be definitely appreciated.

what about `cueDisjunctionDef` ?

Reply:
oups forgot to put a proper name :x will think about it